pkgs/applications/graphics/veusz/default.nix

```
@@ -25,12 +25,12 @@ python3Packages.buildPythonApplication rec {
     wrapQtApp "$out/bin/veusz"
   '';
 
-  # For some reason, if sip5 is found on the PATH, the option --sip-dir is
+  # For some reason, if sip6 is found on the PATH, the option --sip-dir is
```

```suggestion
  # because the sip path does not end in PyQt5, we cannot use the --sip-dir option to set it
  # see https://github.com/veusz/veusz/blob/53b99dffa999f2bc41fdc5335d7797ae857c761f/pyqtdistutils.py#L298
```

The comment is referencing sip5, not sip6, so if we change to sip4 it will no longer apply.

We still need the patch, but for other reasons: The sip dir does not end in PyQt5.